.tool-versions

```diff
@@ -1,3 +1,4 @@
-ruby 2.6.5
-yarn 1.12.3
-nodejs 12.15.0
+ruby 2.7.2
+yarn 1.22.5
+nodejs 12.18.4
```

Comment:
https://github.com/DFE-Digital/publish-teacher-training/blob/master/.nvmrc was missed. I suggest we drop this in favour of asdf